src/app/query/bitcoin/transaction/use-check-utxos.ts

```diff
@@ -0,0 +1,57 @@
+import { useCallback, useState } from 'react';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useBitcoinClient } from '@app/store/common/api-clients.hooks';
+import { useCurrentNetworkState } from '@app/store/networks/networks.hooks';
+
+export function useCheckInscribedUtxos(txIds: string[]) {
```

Comment:
```suggestion
export function useCheckInscribedUtxos(txids: string[]) {
```

https://trustmachines.slack.com/archives/C05LAP952E7/p1699351436402149